src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -327,6 +338,42 @@
         (rr/not-found (i18n/tru "Could not find environment ''{0}''" environment))))))
 
 (schema/defn ^:always-validate
+  module-info-from-jruby->module-info-for-json  :- EnvironmentModulesInfo
+  "Creates a new map with a top level name that corresponds to the requested environment
+  and a top level key modules which contains the module information obtained from JRuby."
```

Comment:
@gguillotte I think maybe wrapping modules in \` would make that clearer. (i.e. a top level key `modules` which contains....)